docs/src/components/Example.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+import React from 'react';
+import Box from '../../../src/Box/Box';
+import Text from '../../../src/Text/Text';
+import Checkerboard from './Checkerboard';
+import Link from '../../../src/Link/Link';
+import Icon from '../../../src/Icon/Icon';
+import Checkbox from '../../../src/Checkbox/Checkbox';
+import Column from '../../../src/Column/Column';
+import Mask from '../../../src/Mask/Mask';
+import Heading from '../../../src/Heading/Heading';
+import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
+
+type Props = {|
+  defaultCode: string,
+  scope: Object,
+|};
+
+export default ({ defaultCode, scope }: Props) => (
```

Comment:
@chrislloyd I also added `stacked` to `Example` - could we add it back in here?

We use this for the `Container` example which needs a minWidth of `800`

Reply:
Sure thing